src/query/expression/src/expression.rs

```diff
@@ -611,3 +611,11 @@ impl<Index: ColumnIndex> RemoteExpr<Index> {
         }
     }
 }
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, PartialEq, Eq)]
+pub enum FieldDefaultExpr {
```

Comment:
how about use `RemoteExpr` instead, which is more general.

Reply:
good idea, but we need this fix as soon as possible, while I am a little busy now,  maybe we can refactor it later.

Reply:
Since the PR needs are very urgent, we can first merge it and optimize codes later.